packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -198,42 +213,50 @@ fn public_storage_fns(
     };
 
     match &value {
-        Value::Status => value_status_fns(variant, &param_list, &storage_method, key),
+        Value::Status => value_status_fns(
+            &param_list,
+            &storage_method,
+            key,
```

Comment:
key and param_list can be constructed within this impl method as well (and potentially in their own helper method)